### PR TITLE
Added Home Page

### PR DESCRIPTION
**Pull request by sanjitsarkar**
In this PR,
I have added Home Page and configured redux-toolkit
**Home Page**
Screenshot from 2022-05-11 00-33-05

### CODE REVIEW

**Review by vermasavita — COMMENTED** [review]
Netflify Deployment checks are not successful. Please check that.

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,7 +1,54 @@
...
+            onClick={() => setIsNavOpen(true)}
+          />
+        ) : (
+          <MdClose
+            className="cursor-pointer rounded-full p-2 w-10 h-10 shadow-md sm:hidden"
+            onClick={() => setIsNavOpen(false)}
+          />
+        )}
+        <div className="site-title text-xl sm:inline-block hidden ">
+          <img src="" alt="" />
```
Comment (anshulraheja) [inline src/components/Header.jsx]: Remove the empty image tag if you are not gonna use it. 
UI looks awesome. The rest of the code looks good to merge

**Review by anshulraheja — APPROVED** [review]

File: src/components/Header.jsx (hunk 1 above)
Comment (sanjitsarkar) [reply src/components/Header.jsx]: done

**Review by vermasavita — APPROVED** [review]

File: src/utils/index.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+};
+
+export const initialSignupCredState = {
+  email: "",
+  password: "",
+  confirmPassword: "",
+  name: "",
+};
+
+export const initialLoginCredState = { email: "", password: "" };
```
Comment (deekshithmd) [inline src/utils/index.js:144]: UI looks clean and good, went through code couldn't find any suggestions now. You can proceed to merge.

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**